apps/evans/lib/filesize.ts

```diff
@@ -1,12 +1,12 @@
 export function formatFilesize(bytes: number): string {
-  const kilobytes = bytes / 1024;
-  if (kilobytes < 1024) {
+  const kilobytes = bytes / 1000;
+  if (kilobytes < 1000) {
```

Comment:
i didn't realize this, but it seems like almost all OS's communicate to users in terms of 1000 not 1024. realized it by comparing some mp3/m4b file sizes, which i thought were slightly wrong.